packages/components/button/examples/ToggleButtonBasicExample.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from 'react';
+import { ToggleButton } from '@contentful/f36-components';
+import { PreviewIcon } from '@contentful/f36-icons';
+
+export default function ToggleButtonBasicExample() {
+  const [isActive, setIsActive] = useState(false);
+  return (
+    <ToggleButton
+      isActive={isActive}
+      onToggle={() => {
+        setIsActive(!isActive);
+      }}
+      icon={<PreviewIcon />}
+    >
+      Toggle Button
+    </ToggleButton>
```

Comment:
Is it possible to find a real ToggleButton in the webapp and use its label instead of a generic "Toggle Button"?

Reply:
All web app examples are rather weird, so I came up with another example of `Like` button here